src/components/RQRCode/src/index.scss

```diff
@@ -22,5 +22,24 @@
       font-weight: 500;
       color: #ffffff;
     }
+
+    & .ray-qrcode__error-btn {
+      .ray-icon,
+      .n-button__content {
+        color: #fff;
+      }
+      .n-button:hover {
+        .ray-icon,
+        .n-button__content {
+          color: var(--n-text-color-hover);
+        }
+      }
+      .n-button:active {
+        .ray-icon,
+        .n-button__content {
+          color: var(--n-text-color-pressed);
```

Comment:
Hello, thanks for your PR

After looking at it, I found that the code may need to be re-optimized a bit, which causes the color to be wrong when the button is pressed

<img width="193" alt="image" src="https://github.com/XiaoDaiGua-Ray/ray-template/assets/51957438/f9eed812-f502-4536-bd86-b1724caf13be">